modules/checkbox/react/lib/Checkbox.tsx

```diff
@@ -12,7 +12,7 @@ import {SystemIcon} from '@workday/canvas-kit-react-icon';
 import {checkSmallIcon} from '@workday/canvas-system-icons-web';
 import uuid from 'uuid/v4';
 
-export interface CheckboxProps extends React.InputHTMLAttributes<HTMLInputElement> {
+export interface CheckboxProps extends Themeable, React.InputHTMLAttributes<HTMLInputElement> {
   checked: boolean;
```

Comment:
Shouldn't the checkmark icon flip?

Reply:
I did some research and it seems it's not necessary to flip checkmark in RTL.